### PR TITLE
DirectX9 Grabber

### PR DESCRIPTION
**Pull request by Paulchen-Panther**
**Summary**

The title says it all.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of web configuration, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing setups:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's body (e.g. `Fixes: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature
- [ ] Related documents have been updated (docs/docs/en)
- [ ] Related tests have been updated

**PLEASE DON'T FORGET TO ADD YOUR CHANGES TO CHANGELOG.MD**
- [ ] Yes, CHANGELOG.md is also updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ endif()
 SET ( DEFAULT_AMLOGIC                     OFF )
 SET ( DEFAULT_DISPMANX                    OFF )
 SET ( DEFAULT_OSX                         OFF )
+SET ( DEFAULT_QT                          ON  )
```
Comment (DonaldTrump-2020) [inline CMakeLists.txt:42]: i see you put double SET ( DEFAULT_QT                          ON  )

**Review by DonaldTrump-2020 — COMMENTED** [review]
i have to change from RGB to BGR to correct colororder rgb. byteRGB wizard cannot fix it!

File: CMakeLists.txt (hunk 1 above)
Comment (Paulchen-Panther) [reply CMakeLists.txt:42]: Thanks for Report. 👍

Comment (DonaldTrump-2020) [reply CMakeLists.txt:42]: > Thanks for Report. 👍

you are welcome. And what's about BGR with Direct9X to correct byteRGB?

**Comment by Paulchen-Panther** [comment]
> i have to change from RGB to BGR to correct colororder rgb. byteRGB wizard cannot fix it!

@DonaldTrump-2020 
can you show me the code where you changed the order from rgb to bgr? is the image preview displayed correctly on the webUi or have the colors been swapped? a screenshot would help me a lot

**Comment by DonaldTrump-2020** [comment]
Capture
yes, here you are.

**Comment by Paulchen-Panther** [comment]
> Capture
> yes, here you are.

1. What the hell does the LED byte sequence have to do with the DirectX Grabber?
2. You didn't answer my other questions.
3. Without further details, there is nothing I can do for you.

**Comment by DonaldTrump-2020** [comment]
I meant when i use directx9 grabber, i have to change this byteRGB from RGB to BGR to correct color order.

With QT grabber on windows, i can still use RGB for color order, but with directX9 can not

**Comment by Paulchen-Panther** [comment]
@DonaldTrump-2020 
I corrected the color sequence. Next time just post a screenshot as requested and answer all the questions asked before. Then it is also quicker.

**Comment by DonaldTrump-2020** [comment]
Thank you bro. I will do it next time

**Pull request merged**